File: ayesha/Basics of Programming/Function-And-Arrays/inverted-bar-chart.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    int n;
    cin >> n;
    int arr[n];
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    // write your code here
    int max = arr[0];
    for (int i = 1; i < n; i++)
    {
        if (arr[i] > max)
        {
            max = arr[i];
        }
    }
    for (int floor = 1; floor <= max; floor++)
    {
        for (int j = 0; j < n; j++)
        {
            if (arr[j] >= floor)
            {
                cout << "*\t";
            }
            else
            {
                cout << "\t";
            }
        }
        cout << "\n";
    }
    return 0;
}

File: V.Vennela/Basics of Programming/Functions and Arrays/InvertedBarChart.java
import java.io.*;
import java.util.*;

public class InvertedBarChart{

  public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int a[] = new int[n];
    for(int i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
    }
    int max=a[0];
    for(int i=0;i<n;i++)
    {
        if(a[i]>max)
            max=a[i];
    }
    for(int i=0;i<max;i++)
    {
        for(int j=0;j<n;j++)
        {
            if(a[j]!=0)
            {
                System.out.print("*\t");
                a[j]--;
            }
            else
            {
                System.out.print("\t");
            }
        }
        System.out.println();
    }
  }

}
